default_config.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
We should comment on what these inputs do and their default/allowable values

Reply:
apparent .json doesnt use comments.